openscm/scmdataframe/base.py

```diff
@@ -194,7 +194,7 @@ def _format_wide_data(df):
                 try:
                     try:
                         # most common format
-                        datetime.datetime.strptime(i, "%Y-%m-%d %H:%M:%S")
+                        pd.to_datetime(i, format="%Y-%m-%d %H:%M:%S")
```

Comment:
hi @dashuber, we currently avoid using `pd.to_datetime` because of its time bounds limitations (see https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#timestamp-limitations). It means you get errors like the below for time ranges we want to use

```python
>>> import pandas as pd
>>> pd.to_datetime("2500-02-10 12:00:00")
pandas._libs.tslibs.np_datetime.OutOfBoundsDatetime: Out of bounds nanosecond timestamp: 2500-02-10 12:00:00
```